docs/ADRs/0014-written-language.md

```diff
@@ -0,0 +1,55 @@
+# ADR-0014 Written Language
+
+## Context
+
+In order to expand our reach and foster international collaboration in the field of AI Validation, we have decided to
+conduct all communication in English on public platforms such as GitHub. This decision aims to facilitate better
+understanding and participation from our global colleagues. However, within the Dutch Government, the norm is to
+communicate in Dutch for internal purposes. This ADR will provide  guidelines on which language to use for different
+types of communications.
+
+## Assumptions
+
+There is no requirement to use Dutch as the primary language for all our activities while working for the Dutch
+government. More information can be found in the [More Information](#more-information) section.
+
+## Decision
+
+The following channels will utilize English:
+
+- GitHub Projects
+- GitHub Repository
+- Email to International partners
+
+The primary language for the following channels will be Dutch:
+
+- Mattermost (Internal Collaboration tool)
+- Email to Internal parties
+- Official Internal Documents like memo's or notes to the chamber
+- Guides for Dutch Users on how to use the tools
+- UI for the tools we will make
```

Comment:
Tweede Kamer in English is House of Representatives: https://en.wikipedia.org/wiki/House_of_Representatives_(Netherlands)

```suggestion
- Mattermost (internal collaboration tool)
- Email to internal parties
- Official Internal documents like memo's or notes to the house of representatives
- Guides for Dutch users on how to use the tools
- UI for the tools we will make
```